Unconscious condition

rosebrallier opened the pull request:
Also updated blacking out to result in Unconsciousness rather than Stun re Issue #694

rosebrallier committed:
Unconscious condition

Also updated blacking out to result in Unconsciousness rather than Stun

Turtlelord26 reviewed (commented):
Can we also mention that the unconscious character is unaware of their surroundings?

bleehu reviewed (approved):
Agree with :turtle:  on unaware, would also like to add unable to speak. But I also feel that is very strongly implied by the title of the condition.

rosebrallier commented:
> also like to add unable to speak. But I also feel that is very strongly implied by the title of the condition.
I consider speaking to be included under "can take no actions"- Free actions are still actions.

rosebrallier committed:
Update 01_05_Conditions.txt

Added unaware

rosebrallier committed:
Update 01_02_Combat_Rules.txt

Edited Nonlethal to account for official Unconscious condition.
Capitalized a few instances of Unconscious throughout the doc
And fixed an instance of "do damage"
Scope creep- fixed skills typography in Maneuvers
More Scope creep- fixed some typography for skills and stats
